pom.xml

```diff
@@ -126,6 +126,7 @@
     <helix.version>1.0.4</helix.version>
     <zkclient.version>0.7</zkclient.version>
     <jackson.version>2.12.7</jackson.version>
+    <jackson-databind.version>2.12.7.1</jackson-databind.version>
```

Comment:
This version exists only for Jackson Databind: 2.12.7.1 is not a version that applies to other Jackson modules or submodules.